### PR TITLE
Update fluent_validation to 9.2.0.

### PR DESCRIPTION
**Pull request by ShadyNagy**
Update fluent validation to 9.2.0.

### CODE REVIEW

File: src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <IsPackable>false</IsPackable>
+      <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (tmarkovski) [inline src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj:4]: This is a packable project, should target netstandard.

File: src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (tmarkovski) [inline src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj:4]: This is a packable project, should target netstandard.

File: src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (tmarkovski) [inline src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj:3]: This is a packable project, should target netstandard.

File: src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj (hunk 1 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj:4]: If it is a packable project then why IsPackable set to false or you mean somthing else by packable?
fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

File: src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj (hunk 2 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj:4]: fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

File: src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj (hunk 3 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj:3]: fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

File: src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj (hunk 1 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj]: Hyperledger.Aries.AspNetCore removed from Hyperledger.Aries.TestHarness as it is not used.

File: src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj (hunk 2 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.Routing.Mediator/Hyperledger.Aries.Routing.Mediator.csproj]: Hyperledger.Aries.AspNetCore removed from Hyperledger.Aries.Routing.Mediator and used Microsoft.AspNetCore.Http.Abstractions instead.

File: src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj (hunk 3 above)
Comment (ShadyNagy) [reply src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj:3]: It is used in this file MvcBuilderExtensions.cs.

**Review by tmarkovski — DISMISSED** [review]

**Review by tmarkovski — DISMISSED** [review]

**Comment by tmarkovski** [comment]
Force push messes up travis ci progress reporting. Please do another commit, without `git push`.

**Comment by ShadyNagy** [comment]
Where is the problem?

image

**Comment by tmarkovski** [comment]
@ryjones It seems Travis isn't always reporting back the status. I tried restarting the build on some occasions, but it didn't work. Can you help us with this?

**Comment by ryjones** [comment]
Do you want me to merge the change?

**Comment by ShadyNagy** [comment]
@tmarkovski now all should be fine.

**Review by tmarkovski — APPROVED** [review]

**Review by tmarkovski — APPROVED** [review]

**Pull request merged**